static/app/views/replays/replayTable.tsx

```diff
@@ -0,0 +1,193 @@
+import React, {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import Duration from 'sentry/components/duration';
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import UserBadge from 'sentry/components/idBadge/userBadge';
+import Link from 'sentry/components/links/link';
+import Placeholder from 'sentry/components/placeholder';
+import TimeSince from 'sentry/components/timeSince';
+import {IconCalendar} from 'sentry/icons';
+import space from 'sentry/styles/space';
+import {NewQuery, PageFilters} from 'sentry/types';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {generateEventSlug} from 'sentry/utils/discover/urls';
+import getUrlPathname from 'sentry/utils/getUrlPathname';
+import theme from 'sentry/utils/theme';
+import {useLocation} from 'sentry/utils/useLocation';
+import useMedia from 'sentry/utils/useMedia';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+import withPageFilters from 'sentry/utils/withPageFilters';
+
+import indexReplayEntriesById from './utils/indexReplayEntriesById';
+import {Replay} from './types';
+
+type Props = {
+  replayList: Replay[];
+  selection: PageFilters;
+};
+
+export type ReplayDurationAndErrors = {
+  count_if_event_type_equals_error: number;
+  'equation[0]': number;
+  id: string;
+  max_timestamp: string;
+  min_timestamp: string;
+};
+
+function ReplayTable(props: Props) {
+  const location = useLocation();
+  const organization = useOrganization();
+  const {projects} = useProjects();
+  const {replayList} = props;
+  const isScreenLarge = useMedia(`(min-width: ${theme.breakpoints[0]})`);
+
+  const getEventView = () => {
+    const {selection} = props;
+    const query = replayList.map(item => `replayId:${item.id}`).join(' OR ');
+    const eventQueryParams: NewQuery = {
+      id: '',
+      name: '',
+      version: 2,
+      fields: [
+        'replayId',
+        'project',
+        'max(timestamp)',
+        'min(timestamp)',
+        'equation|max(timestamp)-min(timestamp)',
+        'count_if(event.type,equals,error)',
```

Comment:
`ReplayDurationAndErrors` doesn't look like it matches the fields here.

`id` in the type, but `replayId` in the query
also `project` is missing from the type.